packages/dialect-react-ui/components/Broadcast/BroadcastForm.tsx

```diff
@@ -174,6 +175,11 @@ function BroadcastForm({ dapp }: BroadcastFormProps) {
   }
 
   const sendBroadcastMessage = async () => {
+    if (noUsers) {
+      setError(new Error('No users in the audience for this broadcast'));
```

Comment:
does it makes sense to store error but not just error msg?

Reply:
Hm.I do store error because I assume there could be some useful info in case of the real error in underlying sdk method occurred. However, I can't come up with some use case right now;

I could change to store only message. Wdyt?